src/style.css

```diff
@@ -0,0 +1,96 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(./fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir-bold;
+  font-weight: bold;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  color: #3d4e62;
+  font-weight: 300;
+  font-size: 16px;
+  --input-height: 70px;
+  --padding-left: 62px;
+  --padding-top: 25px;
+  --background-position: 25px 25px;
+}
+
+.search__bar {
+  margin-top: 20px;
+}
+
+.search__bar-input {
+  font-family: Avenir, sans-serif;
+  font-size: 1rem;
+  line-height: 22px;
+  box-sizing: border-box;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: var(--background-position);
+  background-size: 19px 19px;
+  width: 100%;
+  height: var(--input-height);
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  padding-left: var(--padding-left);
+}
+
+.search__bar-input::placeholder {
+  padding-bottom: 26px;
+  padding-top: var(--padding-top);
+  color: #3d4e61;
+}
+
+.search__bar-input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  padding-left: var(--padding-left);
+}
+
+.search__bar-input:focus {
+  background-color: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  filter: drop-shadow(0 4px 4px rgba(0, 0, 0, 0.25));
+  font-weight: bold;
+  border: 1px solid #e1e7ed;
+  padding-left: var(--padding-left);
+  outline: none;
+}
+
+.search__bar--small {
+  margin-top: 20px;
+}
+
+.search__bar-input--small {
+  font-family: Avenir, sans-serif;
+  line-height: 20px;
+  font-size: 14px;
+  box-sizing: border-box;
+  width: 100%;
+  height: calc(var(--input-height) / 1.66);
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
```

Comment:
DRY - remove styles that have the same values as in the `.search__bar-input` selector